spec/controllers/facility_account_users_controller_spec.rb

```diff
@@ -215,6 +215,36 @@
             do_request
           end
         end
+
+        context "from anything to owner" do
```

Comment:
Can you add this as a simple system spec for this instead?  I should have mentioned this earlier but we are moving away from controller specs in the codebase.